.github/workflows/_01_pre_check.yml

```diff
@@ -84,6 +84,9 @@ jobs:
       - name: Install pnpm 💿
         run: npm install -g pnpm
 
+      - name: Install typescript compiler 📦
+        run: npm install -g typescript
+
```

Comment:
```suggestion
```

Reply:
why is this not required? i needed it installed separately on my machine